index.js

```diff
@@ -0,0 +1,183 @@
+var petsweep = new PictureSelect('petsweep', 12, 'img/pet-sweep.jpg');
+var scissors = new PictureSelect('scissors', 13, 'img/scissors.jpg' );
+var shark = new PictureSelect('shark', 14, 'img/shark.jpg');
+var sweep = new PictureSelect('sweep', 15, 'img/sweep.png');
+var tauntaun = new PictureSelect('tauntaun', 16, 'img/tauntaun.jpg');
+var unicorn = new PictureSelect('unicorn', 17, 'img/unicorn.jpg');
+var usb = new PictureSelect('usb', 18, 'img/usb.gif');
+var watercan = new PictureSelect('watercan', 19, 'img/water-can.jpg');
+var wineglass = new PictureSelect('wineglass', 20, 'img/wine-glass.jpg');
+var randomPics = [];
+var oldRandomPics = [];
+var totalClicks = 0;
+var clickLimit = 25;
+var tableEl = document.getElementById('pic-table');
+var rowEl = document.createElement('tr');
+tableEl.appendChild(rowEl);
+var fieldEl = document.createElement('td');
+var picturesAll = [
+  bag,
+   banana,
+    bathroom,
+     boots,
+      breakfast,
+       bubblegum,
+        chair,
+         cthulhu,
+          dogduck,
+           dragon,
+            pen,
+             petsweep,
+              scissors,
+               shark,
+                sweep,
+                 tauntaun,
+                  unicorn,
+                   usb,
+                    watercan,
+                     wineglass
+                   ];
+
+var chooseRandomPic = function() {
+  var randomPic = picturesAll[Math.floor(Math.random() * picturesAll.length)];
+  // console.log(randomPic);
+  return randomPic;
+};
+
+chooseRandomPic();
+
+var threeRandomPics = function(){
+  console.log('!!!!!!!!!!!!!!!!!!!');
+  console.log('threeRandomPics,   oldRandomPics: ', oldRandomPics);
+  for (var i = 0; i < 3; i++) {
+    console.log(randomPics);
+    var randomPicture = chooseRandomPic();
+    console.log('randomPicture: ', randomPicture);
+    // if(i === 0) {
+    //   randomPics.push(randomPicture);
+      // console.log(randomPics);
+    // }else if(i !== 0) {
+    if(randomPics.includes(randomPicture) || oldRandomPics.includes(randomPicture)){
+      i--;
+        // console.log(randomPics);
+    }else{
+      randomPics.push(randomPicture);
+        // console.log(randomPics);
+    }
+  }
+};
+
+threeRandomPics();
+
+PictureSelect.prototype.createImage = function(i) {
+  var imageEl = document.createElement('img');
+  imageEl.setAttribute('src', this.imgSource);
+  imageEl.setAttribute('id', this.imgName);
+  imageEl.dataset.index = i;
+  this.views++;
+  return imageEl;
+};
+
+//-------------------------------clickcounter----------------------------
+
+//-------------------------------table----------------------------------
+
+function runCreateTable(){
+  for( var i = 0; i < randomPics.length; i++){
+    var fieldEl = document.createElement('td');
+    fieldEl.appendChild(randomPics[i].createImage(i));
+    rowEl.appendChild(fieldEl);
+  }
+};
+
+runCreateTable();
+
+function timesViewed(){
+
+}
+
+function incrementScores(index){
+  console.log('%&%*%(%)%%(%*&%&%*)');
```

Comment:
A clean way to have a console log that uniquely describes each function is just to have the console log print the function's name as a string, like this: 

```javascript
function incrementScores(index){
  console.log('incrementScores');
}
```